apps/epic-web/src/pages/full-stack-workshop-series-vol-1.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {GetStaticProps} from 'next'
+import {MDXRemoteSerializeResult} from 'next-mdx-remote'
+import serializeMDX from '@skillrecordings/skill-lesson/markdown/serialize-mdx'
+import {sanityWriteClient} from '@skillrecordings/skill-lesson/utils/sanity-server'
+import {getProduct, Product} from 'lib/products'
+import ProductTemplate from 'templates/product-template'
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  const product = await getProduct(
+    'kcd_product-f000186d-78c2-4b02-a763-85b2e5feec7b',
```

Comment:
hard coded this here as a top level URL to expedite since it's a short-lived product anyway